mfr/extensions/image/static/js/jquery.zoom.js

```diff
@@ -53,6 +54,7 @@
                 top: 0,
                 left: 0,
                 opacity: 0,
+                "background-color": "white",
```

Comment:
Do not change the `opacity: 0` in the line above, which is the initial style setting. "0 opacity" enables the fading effect during zoom when it is gradually `.fadeTo()`ed 1 (see [here](https://github.com/CenterForOpenScience/modular-file-renderer/blob/master/mfr/extensions/image/static/js/jquery.zoom.js#L140)).

Reply:
Use pure white since the [default](https://github.com/CenterForOpenScience/modular-file-renderer/blob/master/mfr/extensions/image/settings.py#L8) is white [when](https://github.com/CenterForOpenScience/modular-file-renderer/blob/master/mfr/extensions/image/export.py#L59) other types of images are exported to `.JPEG`. This background setting is applied to all images on the frontend.